app/main.js

```diff
@@ -20,3 +20,11 @@ if (process.env.NODE_ENV !== 'production' || process.env.HAS_STYLEGUIDE) {
 }
 
 window.vm = new Vue(vueOptions);
+
+// Set's current theme id to the vuex store read by url param for e.g. ?theme=01
+const themeId = window.location.search.split('theme=')[1];
```

Comment:
- [ ] You must extract the value by regex because there can be other query params in the future (e.g. endless scroll or filters which will most likely manipulate the url. Maybe you can add the helper method described on this page under `JavaScript Fallback` to create a more sophisticated query handler, which we probably will need in the future anyway (the linked polyfill looks a bit big): https://davidwalsh.name/query-string-javascript